src/examples/example1.obfl

```diff
@@ -168,6 +176,9 @@
 			<block>
 				Page number within style: <style name="strong"><page-number ref-id="a"/></style>
 			</block>
+			<block display-when="!$starts-at-top-of-page">
```

Comment:
This should be `(! $starts-at-top-of-page)`

Reply:
Thanks. Corrected.